openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/3-functional/ZDKG-Backup-Restore-CSI/backup-restore-csi

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+set -x
+
+pod(){
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-konvoy && bash openebs-konvoy-e2e/pipelines/OpenEBS-base/stages/3-functional/ZDKG-backup-restore-csi/backup-restore-csi node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"' '"'$RELEASE_TAG'"' '"'$VELERO_VERSION'"' '"'$VELERO_PLUGIN_VERSION'"'
```

Comment:
Is this PLUGIN VERSIONS added into the CI/CD variable.

Reply:
yes